content/docs/user-guide/experiment-management/cleaning.md

```diff
@@ -0,0 +1,262 @@
+│ workspace               │ -            │      - │ 64               │
+│ cnn-48                  │ 09:11 AM     │ 0.9131 │ 48               │
+│ main                    │ Jul 21, 2021 │ 0.9189 │ 16               │
+│ 8583124                 │ Jul 20, 2021 │ 0.9132 │ 17               │
+└─────────────────────────┴──────────────┴────────┴──────────────────┘
+```
+
+### Keeping experiments in all tags
+
+- [ ] `--all-tags` example
+
+### Keeping experiments in all commits
+
+- [ ] `--all-commits` example
+
+## Removing Experiments in Remotes
+
+As you push the experiments with `dvc exp push`, remotes may be become cluttered
+with experiment references.
+
+DVC doesn't provide a shortcut for cleaning up the experiments in remotes but
+you can use Git plumbing commands to remove experiment references from remotes.
+
+First get the list of experiments with their hash values.
+
+```dvc
+$ git ls-remote origin 'refs/exps/*'
+```
+
+Then we can use `git push -d` as any other Git reference:
+
+```dvc
+$ git push -d origin refs/exps/path/to/ref
+```
+
+If you want to delete **all** experiments in a remote, you can use a loop:
+
+```dvc
+$ git ls-remote origin 'refs/exps/*' | cut -f 2 | while read exppath ; do
+   git push -d origin "${exppath}"
+done
+```
+
+## Deleting Experiment-Related Objects in DVC Cache
+
+Note that `dvc exp gc` and `dvc exp remove` doesn't delete any objects in the
+DVC <abbr>cache</abbr>. In order to remove the cache objects, e.g. model files,
+intermediate artifacts, etc. related with the experiments, you can use `dvc gc`
+command.
+
+## Deleting All Queued Experiments
+
+When you created experiments to be run in the queue with `--queue` option of
+`dvc exp run`, and later decide not to run them, you can remove by either
+`dvc exp gc --queued` or `dvc exp remove --queue`. Both of these commands remove
+_all_ queued experiments.
+
+```dvc
+$ dvc exp run --queue -S param=10
+Queued experiment '7b83744' for future execution.
+$ dvc exp run --queue -S param=20
+Queued experiment '68808d5' for future execution.
+$ dvc exp show
+```
+
+```dvctable
+┏━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━┓
+┃ Experiment   ┃ Created      ┃ param ┃
+┡━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━┩
+│ workspace    │ -            │ -     │
+│ 04abbb7      │ Jul 21, 2021 │ -     │
+│ ├── *68808d5 │ 12:05 PM     │ 20    │
+│ └── *7b83744 │ 12:05 PM     │ 10    │
+└──────────────┴──────────────┴───────┘
+```
+
+You can delete these queued experiments with `dvc exp remove --queue`.
+
+```dvc
+$ dvc exp remove --queue
+$ dvc exp show
+```
+
+```dvctable
+┏━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━┓
+┃ Experiment   ┃ Created      ┃ param ┃
+┡━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━┩
+│ workspace    │ -            │ -     │
+│ 04abbb7      │ Jul 21, 2021 │ -     │
+└──────────────┴──────────────┴───────┘
+```
+
+---
+
+> BELOW is from GS:Experiments, will be removed after review
+
+## Cleaning up
+
+Let's take another look at the experiments table:
```

Comment:
Not clear what this section is about based on it's title. The whole page is about cleaning up exps.

Reply:
TBH I'm not sure what it is about even after reading the contents. Maybe it could be in https://dvc.org/doc/command-reference/exp/show instead as an example that links to this guide for more details on cleaning up exps.

Reply:
"BELOW is from GS:Experiments, will be removed after review" :)